notice_and_comment/static/regulations/css/less/module/nc-homepage-custom.less

```diff
@@ -0,0 +1,419 @@
+    font-size: 40px;
+    line-height: 45px;
+  }
+
+  .hero {
+    background-color: unset;
+    background-image: url(../../regulations/img/homepage-hero.jpg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    border-bottom: none;
+    overflow: auto;
+    padding-top: 30px;
+    width: 100%;
+  }
+
+  .hero-content {
+    max-width: 820px;
+    margin: auto;
+
+    h3 {
+      font-size: 28px;
+      margin: 25px 0;
+    }
+  }
+
+/*
+Proposed Rule Box
+================
+*/
+  .proposed-rule {
+    background-color: @white;
+    border: 1px solid @gray;
+    padding: 25px;
+
+    .comments-date {
+      border-bottom: 1px solid @light_blue;
+      font-size: 18px;
+      margin-bottom: 20px;
+      padding-bottom: 20px;
+
+      .due {
+        float: left;
+      }
+
+      .date {
+        font-size: 15px;
+        float: right;
+      }
+    }
+
+    .agency {
+      color: @dark_blue;
+      .font-bold;
+      display: inline-block;
+      line-height: 45px;
+      font-size: 40px;
+      vertical-align: top;
+      width: 100px;
+    }
+
+    .proposed-rule-name {
+      display: inline-block;
+      .font-bold;
+      font-size: 20px;
+      max-width: 600px;
+      vertical-align: top;
+    }
+  }
+
+  .read-proposed-rule {
+    background-color: @dark_blue;
+    color: @light_gray;
+    font-size: 17px;
+    padding: 7px 7px 7px 125px;
+
+    .fa {
+      padding-left: 5px;
+      vertical-align: middle;
+    }
+
+    a {
+      color: @white;
+    }
+  }
+
+/*
+Call Out Comment after Hero
+================
+*/
+  .call-out {
+    background-color: @light_gray;
+    margin: 60px 0;
+    padding: 30px 0;
+    text-align: center;
+  }
+
+  .call-out-content {
+    .font-light;
+    font-size: 30px;
+    line-height: 35px;
+    margin: auto;
+    max-width: 620px;
+
+    strong {
+      font-weight: 500;
+    }
+  }
+
+  .read-and-write,
+  .about-this-tool {
+    max-width: 820px;
+    margin: 20px auto;
+  }
+
+/*
+Read and write with ease
+================
+*/
+  .read-and-write {
+
+    ul {
+      margin: 50px 0;
+      padding: 0;
+
+      li {
+        margin: 0 0 50px 0;
+      }
+
+      .image {
+        display: inline-block;
+        text-align: right;
+        width: 230px;
+      }
+
+      .content {
+        display: inline-block;
+        margin: 10px 0 0 20px;
+        max-width: 560px;
+        vertical-align: top;
+      }
+
+      h3 {
+        margin-bottom: 0;
+      }
+
+      p {
+        margin-top: 0.5em;
+      }
+
+    }
+
+    .other-ways {
+      background-color: @light_gray;
+      .font-bold;
+      padding: 10px;
+    }
+  }
+
+/*
+Build a good comment
+================
+*/
+  .good-comment {
+    background-color: @light_gray;
+    margin: 50px 0;
+    padding: 80px 0;
+
+    h2 {
+      margin-bottom: 1.2em;
+    }
+
+    ul {
+      padding-left: 0;
+      margin-bottom: 50px;
+    }
+
+    li {
+      margin-bottom: 2em;
+      position: relative;
+
+      .triangle {
```

Comment:
Minor, but why make this triangle in CSS rather than use an image? Images are easier to update and wouldn't require a new dom element (if using `:before`).